include/zephyr/bluetooth/mesh/dfd_srv.h

```diff
@@ -75,6 +87,60 @@ struct bt_mesh_dfd_srv_cb {
 		    const struct bt_mesh_dfu_slot *slot,
 		    const struct bt_mesh_blob_io **io);
 
+	/** @brief Firware upload OOB start callback.
+	 *
+	 *  Called at the start of an OOB firmware upload. The application must
+	 *  start a firmware check using an OOB mechanism, and then call
+	 *  @c bt_mesh_dfd_srv_oob_check_complete. Depending on the return value
```

Comment:
How exactly depending on the return value of that function application makes a decision? It will be more clear, if that aspect is spelt out in doc string. It may be a slight repetition, but it would be useful. 

"If return value is <this>, the application must do <this>, otherwise does <that>."

Reply:
The documentation for `bt_mesh_dfd_srv_oob_check_complete` already explains this. I don't think it's needed twice.